user/firejail/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "firejail"
+pkgver = "0.9.72_git20250221"
+pkgrel = 0
+_commit = "733f9a9c487358ba73b06186ed348f086d8ac839"
+build_style = "gnu_configure"
+configure_args = ["--sysconfdir=/usr/share"]
+configure_gen = []
+make_dir = "."
+make_check_target = "test"
+makedepends = [
+    "gawk",
```

Comment:
gawk is almost surely hostmakedepends, and what's the base-kernel-devel for?

Reply:
base-kernel-devel pulled in linux-headers. Build needs `seccomp.h` (required) and `landlock.h` (optional for landlock support).  scaled this dependency down to linux-headers.  The tests need bash (which was pulled in via base-kernel-devel). Added this as `checkdepends`.